Components/TripPreview/TripPreview.js

```diff
@@ -1,18 +1,51 @@
 import React from 'react';
-import { Text, View } from 'react-native';
+import { Text, View, StyleSheet } from 'react-native';
 
-const TripPreview = ({item}) => {
-  const { name, startDate, endDate, originAbbrev, finalDestinationAbbrev} = item;
+export const TripPreview = ({ route }) => {
+  const { name, startDate, endDate, originAbbrev, finalDestinationAbbrev, description } = route.params;
 
   return (
-    <View>
-      <Text>{name}</Text>
-      <Text>{originAbbrev}</Text>
-      <Text>{finalDestinationAbbrev}</Text>
-      <Text>{startDate}</Text>
-      <Text>{endDate}</Text>
+    <View style={styles.container}>
+      <Text style={styles.name}>{name}</Text>
+      <View style={styles.cities}>
+        <Text style={styles.origin}>{originAbbrev}</Text>
+        <Text style={styles.destination}>{finalDestinationAbbrev}</Text>
+      </View>
+      <View style={styles.cities}>
+        <Text style={styles.startDate}>{startDate}</Text>
+        <Text style={styles.endDate}>{endDate}</Text>
+      </View>
+      <View style={styles.description}>
+        <Text>Notes: {description}</Text>
+      </View>
     </View>
```

Comment:
I have a separate branch with a lot of styling edits. Once I've merged this I'm going to use a rebase approach and integrate the different styling approaches together, then I'll push up another PR.

Reply:
Homepage will just show the trip previews, with a plus button that will take you to the createTrip component. This will be it's own view.
Trips Component will be the overview of all trip information. The user will get to this view by clicking on it's respective TripPreview.

For now, I really like what I'm seeing in the gif, and I'd be happy to tweak it if we do decide to make it match the wireframe a bit more.

Reply:
I feel like the homepage looks best if we just have the Trips, we can add a modal for the form that pops up when you hit an add trip button. 

For the TripPreview I just copied the styling we can make that whatever we want.